packages/manager/src/features/CloudPulse/Alerts/AlertsDetail/AlertDetailChips.tsx

```diff
@@ -0,0 +1,99 @@
+import { Typography } from '@linode/ui';
+import { Grid, useTheme } from '@mui/material';
+import React from 'react';
+
+import { StyledAlertChip } from './AlertDetail';
+
+export interface AlertDimensionsProp {
+  /*
+   * The chips that needs to be displayed
+   */
+  chips: Array<string> | Array<string[]>;
```

Comment:
can we give it a proper name like `values` / `labels`. `Chips` name is not looking good